### DISCUSSION TITLE
Problems with Data/GIS: Possibility of Overlooking Human Experiences

### DISCUSSION THREAD

**Discussion by epham36** [albertkun/21S-ASIAAM-191A#35] [Past Thinking Caps]
A possible problem with using data and relying heavily on data for communicating issues which affect communities or advocating for interventions, is that it may encourage the overquantification/overreliance on numeric measures, while disregarding individual experiences and community knowledge. For instance, transnational issues such as critical refugee studies, often geographical data is used to depict the extent of refugee displacement throughout different regions. While this data is useful to highlight the sheer extent of refugee humanitarian crises, the sheer size of numbers and the dehumanization of people captured in this data fail to properly capture the suffering and human significance of these data. 

Data, when it overlooks the human experiences of people who are captured in the data, can fail to grant pressing topics the true gravitas they deserve, especially when numerical figures grow to be this enormous. 

![image](https://user-images.githubusercontent.com/81584148/113526935-b0cc8000-9570-11eb-9904-3ce07d878b4c.png)

While these data serve as an objective number to depict the scale of displacement in the MENA region (although, its reliability is another issue in and of itself), the true extent of peoples' suffering cannot be captured by a statistic alone. Where oral histories and interviews are common ways of gathering qualitative data about people's experiences more holistically, perhaps there is opportunity for expanded use of methods that better blend GIS data and qualitative data, such as ArcGIS Story Maps.

**Comment by okcirE** [comment]
Totally agree. We definitely cannot rule out personal experiences, culture, and the environment of each people in data reporting. Doing so would lead to inadequacy in results and more harm than good to target populations in need. Definitely, a good reminder to not treat people as a mere study or numerical subjects, and to design studies with a holistic approach.

**Comment by nnhimichelle** [comment]
This is definitely true!! The public may view the data and take it as face value without realizing the personal experiences or other conditions that have lead to certain data patterns. If data gets into the wrong hands, there is a possibility that institutions will make conclusions with it and create decisions that can harm many people. This is definitely a great reminder of how flawed it is when people make conclusions without realizing or even looking into other perspectives.
